### PR TITLE
Small changes

### LINKED ISSUE #273: Square refund API
implement refund API, this may include adding a square receipt token to order entries in mongodb

### LINKED ISSUE #314: Style admin buttons

### LINKED ISSUE #315: Modify email regex and location selection in order form

### PR DESCRIPTION
**Pull request by camwlsn**
Button coloring on orderLiveDisp and SuperAdmin
Changed the email regex to allow 2 character emails
Fixed the placeholder for location selection

close #314 
close #315 
close #273 
We discussed with the prof that we shouldn't be expected to fully implement Square because of security risks and therefore refunds should also be handled by client

### CODE REVIEW

File: MERN Scripts/mern-emsystem/src/App.css
Comment (zachpren0) [inline MERN Scripts/mern-emsystem/src/App.css:1]: Good Change!

File: MERN Scripts/mern-emsystem/src/components/EmployeeLiveDisp.js
Comment (zachpren0) [inline MERN Scripts/mern-emsystem/src/components/EmployeeLiveDisp.js:1]: Nice Changes to Css!

File: MERN Scripts/mern-emsystem/src/components/LocationManagement.js
Comment (zachpren0) [inline MERN Scripts/mern-emsystem/src/components/LocationManagement.js:1]: Good, these buttons needed a refresh

**Review by zachpren0 — APPROVED** [review]
Amazing PR dude

**Review by SpeiserK — APPROVED** [review]
Nice changes to front end.

**Pull request merged**